SourceryTests/ParserSpec.swift

```diff
@@ -178,6 +185,41 @@ class ParserSpec: QuickSpec {
                                 .to(equal([expectedType]))
                     }
                 }
+                
+                context("given typealias") {
+                    it("extracts typealiases properly") {
+                        expect(sut?.parseContents("typealias FooAlias = Foo; class Foo {}", existingTypes: []).1)
+                            .to(equal(
+                                ["FooAlias": "Foo"]
+                            ))
+                    }
+                    
+                    it("replaces variable alias type with actual type") {
+                        let expectedVariable = Variable(name: "foo", type: "FooAlias")
```

Comment:
it's useful to use AAA separation in testing http://wiki.c2.com/?ArrangeActAssert, it improved readability